<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="leftPanel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>前曲率半径R1（mm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>后曲率半径R1（mm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>中心厚度d（mm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>折射率nd:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>阿贝数vd:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>物距L(负值)(mm):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_R2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_D">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_Nd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_Vd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_L">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_R1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>300</width>
       <height>250</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnMode">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>切换</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHelp">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>查看公式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCalc">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>计算分析</string>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="slider_D">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="math_algorithm" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>math_algorithm</class>
   <extends>QWidget</extends>
   <header>math_algorithm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
